### PR TITLE
Test for GetMultiple exchange rates

### PR DESCRIPTION
**Pull request by ghaleroshan**
- [x] Adds test that returns list of rates when rates exits in the db
- [x] Adds test that returns [] rates when there are no rates in the db

### CODE REVIEW

File: test/getMultipleTest.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+const mysqlStub = {};
+
+const exchangeRates = proxyquire(
+    './../src/exchangeRates',
+    { 'namshi-node-mysql': _ => mysqlStub },
+);
+const today = new Date().toISOString().split('T')[0];
+
+describe('exchangeRates', () => {
+    describe('get', () => {
```
Comment (geshan) [inline test/getMultipleTest.js]: please use 2 space tabs not 4.

File: test/getMultipleTest.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                };
+
+                const result = await exchangeRates.getMultiple({});
+                assert.deepStrictEqual(result, []);
+            } catch (err) {
+                console.log(`err`, err);
+                assert.strictEqual(err.message, "should never reach here");
+            }
+        });
+        it("should return employees from the db when employees exists", async () => {
```
Comment (geshan) [inline test/getMultipleTest.js]: there are no employees in this app :)

File: test/getMultipleTest.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+                    rate:1.4374030,
+                    on_date:"2019-06-10T14:00:00.000Z"
+                });
+            } catch (err) {
+                console.log(`err`, err);
+                assert.deepStrictEqual(err.message, "should never reach here");
+            }
+        });
+    });
+});
```
Comment (geshan) [inline test/getMultipleTest.js]: missing empty line.

File: package.json [hunk 4]
```diff
@@ -34,6 +34,7 @@
     "mocha": "^6.2.0",
     "nock": "^10.0.0",
     "nyc": "^14.1.1",
+    "prettier": "1.18.2",
```
Comment (geshan) [inline package.json]: Where is the precommit hook?

File: test/getMultipleTest.js [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+describe('exchangeRates', () => {
+  describe('get', () => {
+    it("should return no rates from the db when no rates exists", async () => {
+      try {
+        mysqlStub.query = (query, params) => {
+          if (query.startsWith(`SELECT from_currency, to_currency`)) {
+            return [];
+            }
+            };
```
Comment (geshan) [inline test/getMultipleTest.js]: Wrong indentation?

File: package.json [hunk 6]
```diff
@@ -31,9 +31,16 @@
     "namshi-node-mysql": "^2.0.0"
   },
   "devDependencies": {
+    "husky": "^3.0.2",
```
Comment (geshan) [inline package.json]: +1

Comment (ghaleroshan) [reply package.json]: Sorry, I don't get it?

**Pull request merged**